### PR TITLE
Remove the blue left margin from Sources tree & fix indentation

### PR DESCRIPTION
**Pull request by MikeRatcliffe**
Associated Issue: # 5235

### Summary of Changes

* Removed bright blue left margin.
* Fixed indentation.

### CODE REVIEW

File: src/components/PrimaryPanes/Sources.css [hunk 1]
```diff
@@ -2,6 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
+:root {
+  --theme-focus-outline: 0 none;
```
Comment (jasonLaster) [inline src/components/PrimaryPanes/Sources.css]: shold we scope it to the source tree?

File: src/components/PrimaryPanes/Sources.css [hunk 2]
```diff
@@ -70,6 +74,10 @@
   display: inline-block;
 }
 
+.sources-list .tree .tree-node:not(.focused):hover {
+  background-color: #204e8a;
```
Comment (jasonLaster) [inline src/components/PrimaryPanes/Sources.css]: is there a good variable?

also should we do it in core?
https://github.com/devtools-html/devtools-core/issues/912

File: src/components/PrimaryPanes/Sources.css (hunk 1 above)
Comment (MikeRatcliffe) [reply src/components/PrimaryPanes/Sources.css]: Done

File: src/components/PrimaryPanes/Sources.css (hunk 2 above)
Comment (MikeRatcliffe) [reply src/components/PrimaryPanes/Sources.css]: Yes, fixed

**Comment by jasonLaster** [comment]
Found a couple of things

1. highlight of watch expressions http://recordit.co/jkOkn1sBZu

2. file icon hover color

**Comment by MikeRatcliffe** [comment]
I have renamed the bug to properly explain what is fixed here... the dotted outline, image hover issues and hover color can all be fixed in core.

**Pull request merged**